src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajaxResult/dto/ResponseDetails.java

```diff
@@ -0,0 +1,34 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajaxResult.dto;
+
+/**
+ * Used as a response for ajax result
+ */
+
+public class ResponseDetails {
```

Comment:
I think I gave you a very bad explanation on what I wanted here.  When you get `HttpServletResponse` as a parameter you can set the response status (`response.setStatus(HttpServletResponse.SC_NOT_FOUND)`) directly on it (or whatever the error you want to give it).  When the axios gets this response it triggers it's `.catch` in which case you can display the error.  [See this link for more information](https://www.baeldung.com/spring-response-status-exception)

Reply:
So this ResponseDetails file isn't required correct?